### DISCUSSION TITLE
Documentation for setting up developer environment

### DISCUSSION THREAD

**Discussion by chris-miner** [mike-lischke/vscode-antlr4#202] [Q&A] [answered]
I used to have a functional VSCode development environment for this project, but had to clean up some things on my development machine.  Anyway, I went looking for one of those project readme.md documents that so often include info about downloading the project, installing related tools, compiling, test running, installing, debugging, etc.  

Any pointers to such a document?

**Comment by mike-lischke** [comment, marked as answer]
Good point Chris. There's no information about how to setup debugging the extension or contributing to it. I have to add such a description yet.
